test/integration/targets/openssl_csr/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Install libffi-devel
```

Comment:
These two first tasks are duplicated in three files (`openssl_csr/tasks/main.yml`, `openssl_privatekey/tasks/main.yml`, `openssl_publickey/tasks/main.yml`): these tasks could be put in a dedicated role, something like `setup_openssl`.